java/src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -359,6 +359,8 @@ void testValid(String testDescription) {
      * Assert that the test with the specified description is valid based on the current model, the supplied hypothesis,
      * and the current validator.
      */
+
+
     void testValidWithHypothesis(String testDescription, @Nullable Model hypothesisModel) {
         assertAndDumpWithHypothesis(testDescription, true, hypothesisModel);
```

Comment:
Do we still need to test the hypothesis model?

Reply:
K.. I removed all Hypo testing.  There are no claimframe tests yet but I have a ticket to handle that next sprint.  LMK if there's anything else.